doc/source/PreConfiguredSites.rst

```diff
@@ -3,149 +3,81 @@
+|                     | Gaea                  | Intel              | ``/ncrc/proj/epic/spack-stack/``                       | EPIC / NOAA-EMC |
+|                     +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+| NOAA (RDHPCS)       | Hera                  | GCC, Intel         | ``/scratch1/NCEPDEV/nems/role.epic/spack-stack/``      | EPIC / NOAA-EMC |
+|                     +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | Jet                   | GCC, Intel         | ``/mnt/lfs4/HFIP/hfv3gfs/role.epic/spack-stack/``      | EPIC / NOAA-EMC |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | Narwhal               | GCC, Intel, oneAPI | ``/p/app/projects/NEPTUNE/spack-stack/``               | NRL             |
+| U.S. Navy (HPCMP)   +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | Nautilus              | Intel              | ``/p/app/projects/NEPTUNE/spack-stack/``               | NRL             |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+| Univ. of Wisconsin  | S4                    | Intel              | ``/data/prod/jedi/spack-stack/``                       | JCSDA           |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+| **Cloud platforms**                                                                                                                         |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | AMI Red Hat 8         | GCC                | ``/home/ec2-user/spack-stack/``                        | JCSDA           |
++ Amazon Web Services +-----------------------+--------------------+--------------------------------------------------------+-----------------+
+|                     | Parallelcluster JCSDA | GCC, Intel         |  *currently unavailable*                               | JCSDA           |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+| NOAA (RDHPCS)       | RDHPCS Parallel Works | Intel              | ``/contrib/spack-stack/``                              | EPIC / JCSDA    |
++---------------------+-----------------------+--------------------+--------------------------------------------------------+-----------------+
+
```

Comment:
This is a much cleaner table.  I assume that the spack-stack wiki will have more information on how to load modules from an environment (i.e. expanding from the top-level spack-stack directory).

Reply:
Yes, please take a look at https://github.com/JCSDA/spack-stack/wiki/spack%E2%80%90stack%E2%80%901.8.0-release-documentation, example Narwhal and Nautilus.

Reply:
Ah, very good.  Thanks!